src/main/java/mate.academy/lib/Component.java

```diff
@@ -1,5 +1,9 @@
 package mate.academy.lib;
 
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+
+@Retention(RetentionPolicy.RUNTIME)
```

Comment:
You can add @Target(ElementType.TYPE) here. TYPE means that it used with classes. But it's not necessery.

Reply:
Thank you Sasha, I'll add it